notebooks/SynthDet_Statistics.ipynb

```diff
@@ -38,21 +38,25 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# import os\n",
-    "# datasetinsights.datasets.unity_perception import download_manifest, Downloader\n",
-    "# data_volume = \"/data\"   # directory where datasets should be downloaded to and loaded from\n",
-    "# run_execution_id = \"ojEawoj\"      # Unity Simulation run-execution-id\n",
-    "# auth_token = \"xxxx\"     # Unity Simulation auth token\n",
-    "# project_id = \"xxxx\"   # Unity Project ID\n",
+    "from datasetinsights.io import create_downloader\n",
+    "#run execution id:\n",
+    "# run_execution_id = \"xxx\"\n",
+    "# #access_token:\n",
+    "# access_token = \"xxx\"\n",
+    "# #annotation definition id:\n",
+    "# annotation_definition_id = \"6716c783-1c0e-44ae-b1b5-7f068454b66e\"\n",
+    "# #unity project id\n",
+    "# project_id = \"xxx\"\n",
+    "# usim_source_uri = f\"usim://{project_id}/{run_execution_id}\"\n",
     "\n",
-    "# data_root = os.path.join(data_volume, run_execution_id)"
+    "# unity_simulation_downloader = create_downloader(source_uri=usim_source_uri, access_token=access_token)\n"
```

Comment:
It's better to make this variable name shorter, Like `dl`.

Reply:
The name here is to distinguish between the http_downloader vs the unity_simulation_downloader.

Reply:
I explicit names like `unity_simulation_downloader` is a better her compared to something that was too generic such as `dl`